test/e2e/set/scyllacluster/datainserter.go

```diff
@@ -0,0 +1,148 @@
+// Copyright (c) 2021 ScyllaDB
+
+package scyllacluster
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"sort"
+
+	"github.com/gocql/gocql"
+	"github.com/scylladb/gocqlx/v2"
+	"github.com/scylladb/gocqlx/v2/table"
+	scyllav1 "github.com/scylladb/scylla-operator/pkg/api/scylla/v1"
+	"k8s.io/apimachinery/pkg/util/rand"
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
+)
+
+const nRows = 10
+
+type DataInserter struct {
+	client            corev1client.CoreV1Interface
+	session           *gocqlx.Session
+	table             *table.Table
+	data              []*test
+	replicationFactor int
+}
+
+type test struct {
+	Id   int
+	Data string
+}
+
+func NewDataInserter(ctx context.Context, client corev1client.CoreV1Interface, sc *scyllav1.ScyllaCluster, replicationFactor ...int) (*DataInserter, error) {
+	rf := 0
+	if len(replicationFactor) > 0 {
+		rf = replicationFactor[0]
+	} else {
+		// Default to a total number of members
+		for _, r := range sc.Spec.Datacenter.Racks {
+			rf += int(r.Members)
+		}
+	}
+
+	session, err := newSession(ctx, client, sc)
+	if err != nil {
+		return nil, err
+	}
+
+	d := &DataInserter{
+		client:  client,
+		session: session,
+		table: table.New(table.Metadata{
+			Name:    "test.test",
```

Comment:
we some times create more inserters on one db, would it be better to generate a UID suffix to avoid possible conflicts in the future?

Reply:
keyspace name is now created from UID